src/clr-angular/button/_buttons.clarity.scss

```diff
@@ -266,12 +342,7 @@
 
   .btn.btn-sm,
   .btn-sm .btn {
-    line-height: $clr-btn-appearance-standard-line-height;
-    letter-spacing: $clr-btn-appearance-standard-letter-spacing;
-    font-size: $clr-btn-appearance-standard-font-size;
-    font-weight: $clr-btn-appearance-standard-font-weight;
-    height: $clr-btn-appearance-standard-height;
-    padding: $clr-btn-appearance-standard-padding;
+    @include btn-sm-appearance();
```

Comment:
I did something similar in my core branch. 😁 I guess I'll be using your mixin after this is merged.